src/containers/CreateDoc/configTutela.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default [
+    {
+        title: "Name",
+        component: <input placeholder="name" />
+    },
+    {
+        title: "Last Name",
+        component: <input placeholder="name" />
+    },
+    {
+        title: "Age",
+        component: <select>
```

Comment:
I don't feel like a jsx component should be hardcoded in a config file, but we can handle that later on, can you just add a TODO to take it out of here ?

Reply:
Yes, sure.